src/components/IconInput/IconInput.js

```diff
@@ -14,6 +14,9 @@ const iconBaseStyles = ({ theme }) => css`
   ${size(theme.spacings.mega)};
   top: 50%;
   transform: translateY(-50%);
+  align-items: center;
+  display: flex;
+  justify-content: center;
```

Comment:
Hm. Is this necessary? We try to avoid `flexbox` when possible because it's [not fully supported](https://caniuse.com/#search=flexbox) in some of the old browsers that we have to support. 😕 

The icon prefix/suffix is already centered, so you should be able to center the currency with appropriate padding or line-height.

Reply:
Yeah, so for this particular icon, it's probably _not_ a blocker right now because our icon input is currently not used in any screens we embed in the mobile apps - but that could easily change.

For example, the Employees screen, Signup, Account, or device purchase.

I don't think we need to sweat about it right now as this is a detail that the end user of the library is not exposed to, but be ready to change this if our requirements change.

Reply:
@connor-baer its need, else its not in the middle, i'm open to other solutions

Reply:
See my previous comment 🙂
> The icon prefix/suffix is already centered, so you should be able to center the currency with appropriate padding or line-height.

Reply:
line-height: 16px; seems to fix it